app/code/Magento/CatalogUrlRewrite/Setup/Patch/Data/UpdateUrlKeyForProducts.php

```diff
@@ -58,7 +58,7 @@ public function apply()
         $table = $this->moduleDataSetup->getTable('catalog_product_entity_varchar');
         $select = $this->moduleDataSetup->getConnection()->select()->from(
             $table,
-            ['value_id', 'value']
+            ['entity_id', 'attribute_id', 'store_id', 'value_id', 'value']
```

Comment:
Hi @ravi-chandra3197,
From the https://github.com/magento/magento2/issues/29365#issuecomment-674852010 - this will not work with Magento Commerce.
I just looked to the sample - here is good example how to detect link field for products:

https://github.com/magento/magento2/blob/f70fef1df9c04e77466977747e7ec62264f2a2f2/setup/src/Magento/Setup/Fixtures/ImagesFixture.php#L310-L315

https://github.com/magento/magento2/blob/f70fef1df9c04e77466977747e7ec62264f2a2f2/setup/src/Magento/Setup/Fixtures/ImagesFixture.php#L532-L543

Could you update your PR accordingly?